web/angular-wallet/src/app/setup/account/account.service.ts

```diff
@@ -56,8 +68,9 @@ export class AccountService {
   }
 
   public getAccountTransactions(id: string, firstIndex?: number, lastIndex?: number, numberOfConfirmations?: number, type?: number, subtype?: number): Promise<TransactionList> {
+    const includeTransactions = semver.satisfies(this.selectedNode.version, '>=2.3.1') || undefined;
```

Comment:
nice. I think you could also `import {satisfies} from 'semver'`, making reducing filesize further :)

Reply:
Not sure,  but I think I tried and didn't work...will check to be sure